### PR TITLE
Suchsaml

### PR DESCRIPTION
**Pull request by fernviridian**
Add configurable params path via ENV variable. 
Add sp remote config template.
Clean up examples from templates.

### CODE REVIEW

File: examples/simplesamlphp/docker-compose.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ php:
 web:
   image: nginx:alpine
   ports:
-    - "8080:80"
+    - "8081:80"
```
Comment (ericmann) [inline examples/simplesamlphp/docker-compose.yml]: I take this to mean you had something else already running on :8080 ... is this change necessary?

**Review by ericmann — APPROVED** [review]

File: examples/simplesamlphp/docker-compose.yml (hunk 1 above)
Comment (fernviridian) [reply examples/simplesamlphp/docker-compose.yml]: @ericmann oops, was from my environment because I use 8080 for lots of things...will revert.
